deepchem/rl/a3c.py

```diff
@@ -21,10 +21,11 @@ def __init__(self, value_weight, entropy_weight, **kwargs):
     self.entropy_weight = entropy_weight
 
   def create_tensor(self, **kwargs):
-    reward, action, prob, value = [layer.out_tensor for layer in self.in_layers]
-    log_prob = tf.log(prob)
-    policy_loss = -tf.reduce_sum(
-        (reward - value) * tf.reduce_sum(action * log_prob))
+    reward, action, prob, value, advantages = [
+        layer.out_tensor for layer in self.in_layers
+    ]
+    log_prob = tf.log(prob + 0.0001)
+    policy_loss = -tf.reduce_sum(advantages * tf.reduce_sum(action * log_prob))
```

Comment:
I don't think that is the definition of advantage.
From the A3C paper
https://arxiv.org/pdf/1602.01783.pdf

![selection_094](https://cloud.githubusercontent.com/assets/1225605/26407104/322a3762-4068-11e7-8093-e5de696d6140.png)

What I implemented here is k=1